guides/common/modules/proc_preparing-a-host-for-external-databases.adoc

```diff
@@ -2,15 +2,10 @@
 = Preparing a Host for External Databases
 
 ifndef::orcharhino[]
-Install a freshly provisioned system with the latest {RHEL} 7 server to host the external databases.
+Install a freshly provisioned system with the latest {RHEL} 8 to host the external databases.
 endif::[]
 ifdef::orcharhino[]
-Install a freshly provisioned system with the latest CentOS 7, Oracle Linux 7, or {RHEL} 7 to host the external databases.
-endif::[]
-
-ifdef::satellite[]
-Subscriptions for Red{nbsp}Hat Software Collections and {RHEL} do not provide the correct service level agreement for using {Project} with external databases.
-You must also attach a {Project} subscription to the base operating system that you want to use for the external databases.
```

Comment:
Why did you remove this block? We need the Satellite sub to get to the Satellite repo to get the `postgresql-evr` package.

I have no idea if the rest of the blob is true too (service level, I doubt it), but the repo/sub is important.